index.html

```diff
@@ -247,7 +247,7 @@ <h2><code>PointerEvent</code> Interface</h2>
     boolean     isPrimary = false;
 };
 
-[Constructor(DOMString type, optional PointerEventInit eventInitDict), Exposed=Window]
+[Constructor(DOMString type, optional PointerEventInit eventInitDict = {}), Exposed=Window]
```

Comment:
I wonder what this means. Where is the default dictionary values that are specified before this line? Is there a way to set the default value to a dictionary with the values specified above? @saschanaz @Ms2ger

Reply:
The `= {}` part should be same as passing JavaScript `{}`, which means those default values are automatically set.

Reply:
i have to admit that i'm lost about the actual meaning of the whole IDL thing ... but I trust @saschanaz you know more what this is about etc (and it does look in line with what some other specs have done to update their IDL)